Artifacts/windows-powershell3/install.cmd

```diff
@@ -0,0 +1,14 @@
+@echo OFF
```

Comment:
I thought about using PS, but batch file seems simpler for the task. Although I did not try, one of the failing condition of this MSU is if you have a powershell window opened. So it might have interfered...
As for downloading, it would have made the script more complex. This MSU is pretty old, I don't think there is much chance of being updated. And I believe it would be a different file name anyway.